spec/models/import_csv_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe ImportCsv, type: :model do
```

Comment:
We mainly use Minitest for tests. Our spec tests are mostly used to generate the api-web documentation. Could you consider using Minitest ?